src/utils.jl

```diff
@@ -102,8 +102,8 @@ it returns
 ```math
 \\sum_{𝐤} |û_{𝐤}|² L_x L_y = \\int u(𝐱)² \\, 𝖽x 𝖽y \\,,
 ```
-where ``û_{𝐤} =`` `uh` ``/(`` `grid.nx` ``e^{- i 𝐤 ⋅ 𝐱₀})``, with ``𝐱₀`` the vector with components
-the left-most position in each direction.
+where ``û_{𝐤} =`` `uh` `` / (n_x e^{i 𝐤 ⋅ 𝐱₀})``, with ``𝐱₀`` the vector with components
+the left-most position in each direction, e.g., for a 2D grid `(grid.x[1], grid.y[1])`.
```

Comment:
I think the second half of this sentence (applies to all instances of this) could still be a little clearer. Maybe instead could be along the lines of
>where ``û_{𝐤} =`` `uh` `` / (n_x e^{i 𝐤 ⋅ 𝐱₀})`` and ``𝐱₀`` is the vector with components in 
the left-most position in each direction, ...

or
>where ``û_{𝐤} =`` `uh` `` / (n_x e^{i 𝐤 ⋅ 𝐱₀})``. The elements of the vector ``𝐱₀`` are the left-most position in each direction, ...

Reply:
Damn I merged before noticing this.

Reply:
No worries! I can do this or just add it to a list and do it in a future PR? It is not completely necessary, only if you think the change makes it clearer.